tools/build_pypi_packages.sh

```diff
@@ -14,7 +14,7 @@ set -x
 # package passed as parameter, can be one of PACKAGES or "all"
 TOBUILD=$1
 # list of packages that can be built and uploaded to pypi
-PACKAGES="wmagent wmcore reqmon reqmgr2 reqmgr2ms global-workqueue acdcserver"
+PACKAGES="wmagent wmagent-devtools wmcore reqmon reqmgr2 reqmgr2ms global-workqueue acdcserver"
```

Comment:
Did you want to specify `wmcorepy3-devtools` here? And I think we should also replace `wmagent` by `wmagentpy3`.

We could discuss whether we should convert all our Py3 based changes to the standard packages that we used to have in Py2, but I guess that can wait for a future opportunity.

Reply:
Hmm, I thought about this. I don't know how I feel about making a PyPI package name with the python version in it. Perhaps we can use these "Classifiers" instead to specify what version of Python is required? https://pypi.org/classifiers/

Doing it now I don't think would cause an issue since we don't use the current py2 PyPI packages for anything (like wmagent).

Reply:
Oh, that's a good point! I somehow thought that this list was referring to the spec names that we have in cmsdist.
If it's only about the packages that we upload to PyPi, I completely agree with you.

In case you need an example of classifiers, here is what I did for the CMSCouchApps PyPi package:
[github.comamaltaro/CMSCouchApps](https://github.com/amaltaro/couchapp/blob/master/setup.py#L31)

Reply:
@goughes Erik, I actually think `wmagent-devtools` will fail to build, since it's not defined in the setup_dependencies.py.
And I also have a question here, what is the difference between `wmagent-devtools` and `wmcore` packages?

Reply:
@amaltaro I added the `wmagent-devtools` package. The `wmcore` package was only focused on source code (python, css, js, etc.). I added some additional directories to the `wmagent-devtools` that I thought would be appropriate. Let me know if you think we should add/remove anything.

Reply:
It looks good to me, Erik! Once we start diving into these PyPi packages, we might extend and/or polish anything that we find. Thanks